packages/client/src/components/Modals/GrantDetails.vue

```diff
@@ -67,7 +67,15 @@
       <b-table
         :items="selectedGrant.interested_agencies"
         :fields="interestedAgenciesFields"
-      />
+      >
+      <template #cell(actions)="row" >
+        <b-row v-if="loggedInUser.email === selectedGrant.interested_agencies[0]['user_email']">
```

Comment:
`selectedGrant.interested_agencies[0]` looks like it's only checking the 1st agency - can we ensure the user won't be found in other agencies? eg. `selectedGrant.interested_agencies[1]` ? Should this comparison be more robust?